CRM/Upgrade/Incremental/sql/4.6.0.mysql.tpl

```diff
@@ -28,12 +28,21 @@ SET name = 'Author Only'
 WHERE option_group_id = @option_group_id_notePrivacy AND value = 1;
 
 --These labels were never translated so just copy them over as names
-UPDATE civicrm_option_value v, civicrm_option_group g
-SET v.name = {localize field='label'}v.label{/localize} 
-WHERE g.id = v.option_group_id AND g.name IN
-('group_type', 'safe_file_extension', 'wysiwyg_editor');
+{if $multilingual}
+  {foreach from=$locales item=locale}
+    UPDATE civicrm_option_value v, civicrm_option_group g
+      SET v.name = v.label_{$locale}
+    WHERE g.id = v.option_group_id AND g.name IN
+    ('group_type', 'safe_file_extension', 'wysiwyg_editor');
+  {/foreach}
```

Comment:
this code snippet will be executed n locale times to update the v.name field. Can't we break the loop after 1st interation?
or can we use 1st locale v.label_{$locales.0} rather interating in foreach?
(https://github.com/civicrm/civicrm-core/blob/master/CRM/Upgrade/Incremental/sql/4.2.beta1.mysql.tpl#L25)